README.rst

```diff
@@ -601,6 +601,10 @@ ROBOTSTXT_OBEY_ is ``True``, or those for sitemaps when using a `sitemap
 spider`_. Certain parameters, like ``browserHtml`` or ``screenshot``, are not
 meant to be used for every single request.
 
+If the ``zyte_api_default_params`` request meta key is set to ``False``, the
+values of the ``ZYTE_API_DEFAULT_PARAMS``, ``ZYTE_API_AUTOMAP_PARAMS`` and
+``ZYTE_API_TRANSPARENT_MODE`` settings for this request are ignored.
```

Comment:
I'm currently ignoring them just in case. I don't think it should make a difference and I expected to have a discussion on this. I agree that as we are currently setting the `zyte_api` meta key only `ZYTE_API_DEFAULT_PARAMS` can make a difference so I'll remove extra code.